website/docs/events/m3.md

```diff
@@ -0,0 +1,48 @@
+# Workshop: Multiplier Event 3
+
+## Schedule
+
+- Date: Thursday, March 2, 2023
+- Time: 10:00-12:00 EEST
+- Place: Online, on Google Meet
+
+### What We Did
+
+> 10:00 - 10:15
+
+We will deliver a short presentation on the status of Operating Systems course.
+We will describe our methodology, the way we store content, how it's presented, how we integrate quizzes and others.
+
+### Opinions from Stakeholders
+
+> 10:30 - 10:45
+
+Participants will be encouraged to voice their opinions on the Operating Systems course based on the previous presentation.
+The stakeholders range from professors to teaching assistants, industry representatives and other content creators.
+
+### Presentation of Metrics
+
+> 10:45 - 11:00
+
+In this section we will present the feedback we have received from students, content creators and other teaching assistants.
+In addition, we will provide an overview of the contributions made by students to our course materials together with an overview of their grades in relation to their activity.
+
+### Open Discussion: 11:00 - 12:00
```

Comment:
No more coffee break? :(

Reply:
@RazvanN7's fault. And coffee isn't healthy anyway.

Reply:
We can add a 10 minute coffee break and make the session only 50 min.